### ISSUE TITLE
Snap Package Support (Feature Request)

### ISSUE THREAD

**Issue by mikeroyal** [drone/drone#2542]
I was thinking in the install and setup could be improved for Ubuntu and other Linux distros by having a [snap package](https://snapcraft.io/) version. If it's not a priority would you mind if I made the snap?

[Crafting your first snap](https://tutorials.ubuntu.com/tutorial/create-your-first-snap#0)

[Build and test snap](https://snapcraft.io/build)

[Getting Started snap template](https://github.com/snapcrafters/fork-and-rename-me)

**Comment by bradrydzewski** [comment]
absolutely, would love help creating a snap. Let's discuss at gitter.im/drone/drone

**Comment by bradrydzewski** [comment]
@mikeroyal just wanted to check and see if you are still interested in this? thanks!

**Comment by mikeroyal** [comment]
Hi @bradrydzewski, Here's the repo for the snap for Drone CLI https://github.com/mikeroyal/drone-cli-snap . It needs a little more testing but it's pretty much ready to go. Definitely feel free to build, test, and register the snap yourself. https://snapcraft.io/build

**Comment by mikeroyal** [comment]
Hi @bradrydzewski bradry made a PR https://github.com/drone/drone-cli/pull/122

**Comment by bradrydzewski** [comment]
thanks Mike! I will take a look and let you know if I have any questions
